ice-phishing-detection-bot/README.md

```diff
@@ -0,0 +1,39 @@
+# Ice Phishing Detection Bot
+
+## Description
+
+This bot detects high approvals for a specific account and if any assets are transfered
+
+## Supported Chains
+
+- Ethereum
+- Binance Smart Chain
+- Polygon
+- Arbitrum
+- Optimism
+- Fantom
+
+## Alerts
+
+- ICE-PHISHING-HIGH-NUM-APPROVALS
+
+  - Fired when an address suddenly has a higher amount of approvals
+  - Severity is always set to "low"
+  - Type is always set to "suspicious"
+  - Metadata fields:
+    - FIRST_TRANSACTION_HASH (the transaction that was first found)
+    - LAST_TRANSACTION_HASH (the tranaction that was found last)
+    - ASSETS_IMPACTED (the amount of assets that were impacted)
+
+- ICE-PHISHING-HIGH-NO-APPROVALS
```

Comment:
this is a bit of a confusing alert id as it seems like this one triggers on transfers?